app/config/wp-demo/install.sh

```diff
@@ -36,7 +36,7 @@ fi
 CIVI_TEMPLATEC="${CIVI_FILES}/templates_c"
 CIVI_UF="WordPress"
 
-civicrm_install
+civicrm_install_transitional
```

Comment:
The issue I see is that we call this function and then `civicrm_install_cv` before we activate the plugin so we get fatal errors.  

An ugly fix is to add 
```bash
pushd "$CMS_ROOT" >> /dev/null
wp plugin activate civicrm
wp eval '$c=[civi_wp()->admin, "add_wpload_setting"]; if (is_callable($c)) $c();'
```

before we call `civicrm_install_transitional`

Reply:
I just wanted to say a belated thanks @kcristiano. I was testing a related problem a week ago, and this clue helped a lot. I've been meaning to dig into understand problem and how to bake-in the fix. (Notwithstanding the github bot, the issue is still in the hopper...)

Reply:
OK, I think if we merge https://github.com/civicrm/civicrm-core/pull/30946, then `civirm_install_transitional` and `civicrm_install_cv` will. work better.